File: app/src/main/java/dev/bltucker/nanodegreecapstone/storydetail/injection/StoryDetailFragmentComponent.java
package dev.bltucker.nanodegreecapstone.storydetail.injection;

import dagger.Subcomponent;
import dev.bltucker.nanodegreecapstone.storydetail.StoryDetailFragment;

@Subcomponent(modules = {StoryDetailFragmentModule.class})
@StoryDetailFragmentScope
public interface StoryDetailFragmentComponent {
    void inject(StoryDetailFragment fragment);
}

File: app/src/main/java/dev/bltucker/nanodegreecapstone/storydetail/injection/StoryDetailFragmentComponent.kt
package dev.bltucker.nanodegreecapstone.storydetail.injection

import dagger.Subcomponent
import dev.bltucker.nanodegreecapstone.storydetail.StoryDetailFragment

@Subcomponent(modules = arrayOf(StoryDetailFragmentModule::class))
@StoryDetailFragmentScope
interface StoryDetailFragmentComponent {
    fun inject(fragment: StoryDetailFragment)
}
